redirects.js

```diff
@@ -53,7 +53,19 @@ function redirects() {
     },
     {
       source: "/extensions",
-      destination: "/contractkit",
+      destination: "/solidity-sdk",
+      permanent: false,
+    },
+    {
+      source: "/contractkit",
+      destination: "/solidity-sdk",
+      permanent: true,
+    },
+    //  old (deprecated) routes
+    {
+      source:
+        "/:network/(edition|nft-collection|token|pack|nft-drop|signature-drop|edition-drop|token-drop|marketplace|split|vote)/:address",
```

Comment:
```suggestion
        "/:network/(edition|nft-collection|token|pack|nft-drop|signature-drop|edition-drop|token-drop|split|vote)/:address",
```

manan changed these to fix a bug, just merge this and it's fine